src/Quartz/Impl/AdoJobStore/Common/dbproviders.netstandard15.properties

```diff
@@ -0,0 +1,19 @@
+# Database provider configuration data
+# Core information taken from Spring Framework .NET - All credits to their great work!
+
+# SQL SERVER
+quartz.dbprovider.SqlServer-20.productName=Microsoft SQL Server Core, provider V4.1.0.0 in framework .NET Core 1
```

Comment:
is it dangerous to indicate that this is the same fellow from .NET 2.0 era or are they so equal that can be called SqlServer-20?